File: android/app/src/main/java/com/example/demointegratern/SecondActivity.java
package com.example.demointegratern;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        Button btnBackRNScreen = findViewById(R.id.btnBackRNScreen);
        btnBackRNScreen.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });
    }
}

File: android/app/src/main/java/com/example/demointegratern/SecondActivity.kt
package com.example.demointegratern

import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class SecondActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)
        val btnBackRNScreen = findViewById<Button>(R.id.btnBackRNScreen)
        btnBackRNScreen.setOnClickListener { finish() }
    }
}